src/plugins/noresolver/noresolver.c

```diff
@@ -46,7 +46,7 @@ int elektraNoresolverGet (Plugin * handle ELEKTRA_UNUSED, KeySet * returned ELEK
 
 	Key * root = keyNew ("system/elektra/modules/" ELEKTRA_PLUGIN_NAME, KEY_END);
 
-	if (keyRel (root, parentKey) >= 0)
+	if (keyCmp (root, parentKey) == 0 || keyIsBelow (root, parentKey) == 1)
```

Comment:
Why not keyIsBelowOrSame? Or we remove keyIsBelowOrSame because it is hardly used?

Reply:
I did not realize this function existed. It's even part of the public api, but I always checked the [documentation](https://doc.libelektra.org/api/current/html/group__keytest.html) and since it's not displayed there, I didn't know about it.

> Or we remove keyIsBelowOrSame because it is hardly used?

If we want to provide the minimal amount of necessary functions for 1.0, then removing it is not a bad idea.

Reply:
Weird, can you find out why it is not in the documentation? (Or report an issue about that.)

>  If we want to provide the minimal amount of necessary functions for 1.0, then removing it is not a bad idea.

Yes, the API should be minimal as extending is easy but removal is not possible after 1.0 release.

Reply:
(Or better report the issue first, if we remove it, we do not need to fix the docu.)